insalan/tournament/views.py

```diff
@@ -172,27 +172,37 @@ def get(self, request, primary_key: int):
                 ).data
                 del team_preser["tournament"]
 
+                team_members: list[int] = team_preser["players"] + team_preser["managers"] + team_preser["substitutes"]
+                can_see_payment_status: bool = request.user.is_staff or request.user.id in team_members
```

Comment:
Ideally this would have been handled with a permission of some kind but it's just as good I guess